### ISSUE TITLE
GitHub Actions instead of Travis?

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/buildout.plonetest#71]
Since Travis is giving less free minutes to public repositories and teams, collective projects may need to switch to GitHub Actions. It would be useful if we can come up with a blueprint for that.

**Comment by ale-rt** [comment]
I agree, this is what I did on dexterity.memebrane https://github.com/collective/dexterity.membrane/blob/master/.github/workflows/main.yml
It might not be perfect, but it somehow works.
I was partly inspired by the work done for the zope packages.

**Comment by idgserpro** [comment]
This is supposed [to be temporary](https://docs.travis-ci.com/user/migrate/open-source-repository-migration#q-why-are-some-of-my-queued-jobs-taking-longer-than-usual-to-build), no? Do you have the official Travis announcement about giving less free minutes do public repositories and teams?
